dodo.c

```diff
@@ -324,8 +324,9 @@ struct Instruction * parse_print(char *source, size_t *index){
      */
     if( isdigit(source[*index]) ){
         ret = parse_number(i, source, index);
-        if(ret == 0)
+        if( ret == 0 ){
```

Comment:
thanks for this, I need to write a style guide at some point.
I prefer

``` if( cond ){
    body
}
```

and

``` if( cond ){
    body
} else {
  body
}
```

:+1: